ozaria/site/components/teacher-dashboard/BaseMyClasses/ClassComponent.vue

```diff
@@ -112,30 +145,30 @@ export default {
             <span>Ozaria Dashboard</span>
           </a>
         </li>
-        <li>
+        <li v-if="me.isCodeNinja() && equinoxTournament.clan">
           <!-- TODO: specific tournament link -->
           <a
             v-tooltip.top="{
               content: 'View tournament leaderboards or play the Equinox arena (days 3-4).',
               classes: 'teacher-dashboard-tooltip lighter-p',
               autoHide: false
             }"
-            href="https://codecombat.com/play/ladder/equinox/clan/65fa5bc654652e2ad959548e?tournament=65fcadac32f2005645fba16b"
+            :href="`/play/ladder/equinox/clan/${equinoxTournament.clan}?tournament=${equinoxTournament._id}`"
```

Comment:
It needs to link to codecombat.com when it's on the ozaria.com version of the teacher dashboard.

Reply:
oh okay, then do that. anyway i scope it for codeninja, so no other usage